hallux/targets/gitlab_suggestion.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, mr_url: str | None):
 
         request_url: str = f"{base_url}/projects/{project_name}/merge_requests/{mr_iid}/changes?unidiff=true"
         headers = {"PRIVATE-TOKEN": f"{os.environ['GITLAB_TOKEN']}"}
-        mr_response = requests.get(request_url, headers=headers)
+        mr_response = requests.get(request_url, headers=headers, verify=False)
```

Comment:
Enable server certificate validation on this SSL/TLS connection.
```suggestion
        mr_response = requests.get(request_url, headers=headers, verify=True)
```